docs/library/axi_adaq8092/index.rst

```diff
@@ -197,19 +197,20 @@ The example design uses a processor to program all the registers. If no
 processor is available in your system, you can create your own IP starting from
 the interface module.
 
-Software Guidelines
+Software Support
 --------------------------------------------------------------------------------
 
-The software for this IP can be found as part of the ADAQ8092 Native FMC Card
-Reference Design at :git-no-OS:`no-OS Software <projects/adaq8092>` and
-Linux is supported at :git-linux:`/`.
+* No-OS project at :git-no-OS:`drivers/adc/adaq8092`
+* No-OS device driver at  :git-no-OS:`drivers/adc/drivers/adc/adaq8092/adaq8092.c`
+* Linux device driver at :git-linux:`drivers/iio/adc/adaq8092.c`
+* Linux device tree at :git-linux:`arch/arm/boot/dts/zynq-zed-adv7511-adaq8092.dts`
+* Linux device driver documentation
+  :dokuwiki:`on wiki <resources/tools-software/linux-drivers/iio-adc/adaq8092>`
 
 References
 -------------------------------------------------------------------------------
 
 * :git-hdl:`library/axi_adaq8092`
```

Comment:
This change applies to all files where a project and a project documentation is available/will be available:
(I know the reference adaq8092_fmc doesn't exist yet, but it will)

```suggestion
* HDL IP core at :git-hdl:`library/axi_adaq8092`
* HDL project at :git-hdl:`projects/adaq8092_fmc`
* HDL project documentation at :ref:`adaq8092_fmc`
```